index.html

```diff
@@ -1929,224 +1974,71 @@ <h2>
 automatically typed to provide stronger type guarantees for use cases that
 require them.
         </li>
-        <li>
-The <code>@protected</code> properties feature of JSON-LD 1.1 is used to ensure
-that terms defined by this specification cannot be overridden. This means that
-as long as the same <code>@context</code> declaration is made at the top of a
-<a>DID document</a>, interoperability is guaranteed between implementations
-that use a JSON-LD processor and implementations that do not.
-        </li>
       </ul>
-
       <section>
-        <h2>
-  Contexts
-        </h2>
-
-        <p>
-  When two software systems need to exchange data, they need to use terminology
-  and a protocol that both systems understand. The <code>@context</code>
-  property ensures that two systems operating on the same DID document are using
-  mutually agreed terminology.
-        </p>
+        <h2>Production</h2>
 
         <p>
-  <a>DID documents</a> MUST include the <code>@context</code> property.
-        </p>
-
-        <p class="note" title="The JSON-LD Context">
-  More information about the
-  <a href="https://www.w3.org/TR/json-ld/#the-context">JSON-LD Context</a> in
-  general can be found in the [[JSON-LD]] specification.
+The <a>DID document</a> is serialized following the rules in the JSON processor, with
+one addition: <a>DID documents</a> MUST include the <code>@context</code> property.
         </p>
 
         <dl>
           <dt><dfn>@context</dfn></dt>
           <dd>
   The value of the <code>@context</code> property MUST be one or more <a>URIs</a>,
-  where the value of the first <a>URI</a> is
-  <code>https://www.w3.org/ns/did/v1</code>. If more than one <a>URI</a> is
-  provided, the <a>URIs</a> MUST be interpreted as an ordered set. It is
-  RECOMMENDED that dereferencing each <a>URI</a> results in a document containing
-  machine-readable information about the context.
+  where the value of the one <a>URI</a> is
+  <code>https://www.w3.org/ns/did/v1</code>. All members of the <code>@context</code>
+  property MUST exist be in the DID properties extension registry. 
```

Comment:
This is about processing globally interoperable documents. This is a MUST.

Reply:
Specifically, the pull request already addresses this with the requirement:

> Deployments of this specification MAY use a custom agreed-upon representation.

I would argue that a representation that uses unknown properties is using its own representation and is outside the scope of this standard.

Reply:
It's exactly because you put that text in there that I suggested "SHOULD" instead of "MUST"... because the spec allows the use of custom agreed-upon representations, and `@context` makes it easy to do that w/o having to use a registry.

Please raise and issue and mark the "MUST vs. SHOULD" as a point of contention that the WG is currently debating (and point to the issue). Let's avoid a perma-thread on the whole MUST vs. SHOULD debate wrt. entries in the registry by just noting this as an open issue and moving on.

Reply:
If we're going to raise it as an issue, I argue that we need to start with `MUST` and raise an issue to lower it to `SHOULD` if the WG agrees, for the reasons above. I strongly believe this section needs to have strong normative requirements and as few SHOULD/MAY portions as possible.